docs/source/usage.rst

```diff
@@ -305,6 +305,21 @@ Input yaml file:
     >>> print(conf.user.home)
     /home/omry
 
+It is possible to set a default value if the environment variable is missing. 
+In the following example, we set '1234' as the default value if the environment
+variable 'DB_PASSWORD' is not defined.
```

Comment:
```suggestion
You can specify a default value to use in case the environment variable is not defined.
The following example sets `12345` as the the default value for the `DB_PASSWORD` environment variable.
```